contracts/IrrevocableVesting.sol

```diff
@@ -83,11 +83,7 @@ contract IrrevocableVesting is IIrrevocableVesting {
             "Balance is not vesting amount"
         );
         beneficiary = beneficiary_;
-        vesting = Vesting({
-            startTimestamp: startTimestamp,
-            endTimestamp: endTimestamp,
-            amount: amount
-        });
+        vesting = Vesting(startTimestamp, endTimestamp, amount);
```

Comment:
This is dangerous because it's easy to mistake the order of fields and the compiler won't help if the types match by coincidence